Sources/FluentUI_iOS/Components/BottomSheet/BottomSheetController.swift

```diff
@@ -696,6 +719,14 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
             stackView.bottomAnchor.constraint(equalTo: bottomSheetContentView.bottomAnchor)
         ])
 
+        if let headerContentView {
+            resizingHandleContentOverlapConstraints = headerContentView.topAnchor.constraint(equalTo: resizingHandleView.bottomAnchor, constant: -currentResizingHandleHeight)
```

Comment:
why not just constrain to top anchor instead of constraining to bottom anchor and then applying negative offset?

Reply:
Also the resizing view should just be added as a subview of the `bottomSheetContentView`. It shouldn't matter what's in the content, just that we overlay on top. And then depending on the bool value, we either shift the content stack down by the resizing handle height, or not. No constraint activating / deactivating needed.